common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -281,6 +282,15 @@ def _is_in_bulk_operation(self, course_key, ignore_case=False):
         """
         return self._get_bulk_ops_record(course_key, ignore_case).active
 
+    def send_bulk_published_signal(self, bulk_ops_record, course_id):
+        """
+        Sends out the signal that items have been published from within this course.
+        """
+        signal_handler = getattr(self, 'signal_handler', None)
```

Comment:
In what situations is `signal_handler` not defined? Can we just initialize it to `None` in a constructor somewhere if that's the case, instead of relying on `getattr`?